handlers/main.yml

```diff
@@ -2,19 +2,9 @@
 - name: Reload systemd units
   become: true
   systemd:
-    name: "{{ restart_item.dest | basename }}"
+    name: "{{ reload_item.dest | basename }}"
     daemon_reload: true
-  with_items: "{{ restart_units.results }}"
+  with_items: "{{ reload_units.results }}"
   loop_control:
-    loop_var: restart_item
+    loop_var: reloadrestart_item
```

Comment:
=> reload_item

Reply:
Blimey, what a stupid mistake. I'm on it.

Reply:
fixed